Documentation/class-libraries.md

```diff
@@ -0,0 +1,46 @@
+.NET Class Libraries
+====================
+
+Class libraries are the [shared library](http://en.wikipedia.org/wiki/Library_(computing)#Shared_libraries) concept for .NET. They enable you to componentize useful functionality into modules that can be used by multiple applications. They can also be used as a means of loading functionality that is not needed or not known at application startup. 
+
+There are three types of class libraries that you can use:
+
+- **Platform-specific** class libraries have access to all the APIs in a given platform (e.g. .NET Framework 4.6, Xamarin iOS), but can only be used by apps and libraries that target that platform.
```

Comment:
Do we really want to explicitly mention ".NET Framework **4.6**" here and throughout the docs? This is bound to get outdated and would need to be updated with each release.